package.json

```diff
@@ -41,9 +41,14 @@
   },
   "husky": {
     "hooks": {
-      "pre-commit": "npm run prettier"
+      "pre-commit": "lint-staged"
     }
   },
+  "lint-staged": {
+    "./{server,src}/**/*.{tsx,ts,js,json,css}": [
```

Comment:
and html?

Currently i just make prettier run on everything it can. Would it be possible to make lint-staged use a wildcard as well? Adding extensions as we go here is probably an uphill battle

Reply:
I guess that might be difficult if lint-staged doesn't know what prettier can and can't process, and the point of this PR is to avoid the full run that `npm run prettier` would do.

Reply:
Actually if it only touches staged files maybe we can just pass * to prettier and let it decide what it can and can't format. Presumably that's still faster than running on everything

Reply:
I'll investigate that!

Reply:
I added a wildcard to the glob now so it should prettify everything it can find and handle